utils/dockerfile_assets/github_dl.py

```
@@ -34,6 +34,30 @@ def validate_binary(binary, checksum, raw_algorithm="sha256") -> bool:
 
+def validate_token(token) -> bool:
+    headers = {
+        "Authorization": f"Bearer {token}",
+        "Accept": "application/vnd.github+json",
+        "X-GitHub-Api-Version": "2022-11-28",
+    }
+
+    response = requests.get("https://api.github.com/rate_limit", headers=headers)
+
+    if response.status_code == 401:
+        print(f"Error: GitHub token is invalid or expired (HTTP 401). Please check your GITHUB_TOKEN.")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)